VCs/ItemsVC.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  ItemsVC.swift
+//  To-Do App
+//
+//  Created by Ananth Desai on 18/02/22.
+//
+
+import UIKit
+
+class ItemsVC: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    var index: Int
+    var itemsTableView: UITableView!
```

Comment:
These can be private? If there is no necessary to expose the property or function, we should declare it as private.